velox/vector/arrow/tests/ArrowBridgeSchemaTest.cpp

```diff
@@ -192,14 +192,15 @@ TEST_F(ArrowBridgeSchemaExportTest, scalar) {
   testScalarType(VARCHAR(), "u");
   testScalarType(VARBINARY(), "z");
 
+  options_.timestampTimeZone = "America/Los_Angeles";
   options_.timestampUnit = TimestampUnit::kSecond;
-  testScalarType(TIMESTAMP(), "tss:");
+  testScalarType(TIMESTAMP(), "tss:America/Los_Angeles");
   options_.timestampUnit = TimestampUnit::kMilli;
-  testScalarType(TIMESTAMP(), "tsm:");
+  testScalarType(TIMESTAMP(), "tsm:America/Los_Angeles");
   options_.timestampUnit = TimestampUnit::kMicro;
-  testScalarType(TIMESTAMP(), "tsu:");
+  testScalarType(TIMESTAMP(), "tsu:America/Los_Angeles");
   options_.timestampUnit = TimestampUnit::kNano;
-  testScalarType(TIMESTAMP(), "tsn:");
+  testScalarType(TIMESTAMP(), "tsn:America/Los_Angeles");
```

Comment:
does Arrow also support timezone offsets? ("+01:00" and such)

Reply:
@pedroerp Yes. I added the "+01:00" timestamp test here.